skk.el

```diff
@@ -591,17 +570,14 @@ dependent."
 	       mode-string-list)))))
 
 (defun skk-make-indicator-alist-1 (mode base)
-  (let* ((ml (if (skk-facep 'mode-line)
-		 'mode-line	  ;GNU Emacs
-	       'modeline))	  ;XEmacs
-	 (string (concat "--" base
-			 (cond ((skk-face-proportional-p ml)
-				":")
-			       ((memq mode '(latin abbrev))
-				"::")
-			       (t
-				":")))))
-    (skk-mode-string-to-indicator mode string)))
+  (skk-mode-string-to-indicator 'mode-line
```

Comment:
This `'mode-line` should be `mode`.
Fixed by https://github.com/skk-dev/ddskk/pull/172